<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::ProxyPropertiesDialog</class>
 <widget class="QDialog" name="Otter::ProxyPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>460</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>titleLineEditWidget</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Otter::LineEditWidget" name="titleLineEditWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="manualConfigurationCheckBox">
         <property name="text">
          <string>Manual</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="manualConfigurationSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="manualConfigurationWidget" native="true">
           <layout class="QGridLayout" name="manualConfigurationLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="2" column="1">
             <widget class="Otter::LineEditWidget" name="httpServersLineEditWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSpinBox" name="httpPortSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QSpinBox" name="httpsPortSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="Otter::LineEditWidget" name="httpsServersLineEditWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QSpinBox" name="socksPortSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QSpinBox" name="ftpPortSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="portLabel">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="protocolabel">
              <property name="text">
               <string>Protocol</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="serverLabel">
              <property name="text">
               <string>Servers</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="ftpCheckBox">
              <property name="text">
               <string>FTP</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="socksCheckBox">
              <property name="text">
               <string>SOCKS5</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="httpCheckBox">
              <property name="text">
               <string>HTTP</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="Otter::LineEditWidget" name="socksServersLineEditWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="httpsCheckBox">
              <property name="text">
               <string>HTTPS</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="Otter::LineEditWidget" name="ftpServersLineEditWidget">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="allCheckBox">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="Otter::LineEditWidget" name="allServersLineEditWidget"/>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="allPortSpinBox">
              <property name="maximum">
               <number>65535</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="automaticConfigurationCheckBox">
         <property name="text">
          <string>Automatic</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="automaticConfigurationWidget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="automaticConfigurationLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="automaticConfigurationSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="automaticConfigurationLabel">
            <property name="text">
             <string>Path to PAC file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Otter::FilePathWidget" name="automaticConfigurationFilePathWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="usesSystemAuthenticationCheckBox">
         <property name="text">
          <string>Use system authentication</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="generalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="exceptionsTab">
      <attribute name="title">
       <string>Exceptions</string>
      </attribute>
      <layout class="QVBoxLayout" name="exceptionsLayout">
       <item>
        <widget class="QLabel" name="exceptionsLabel">
         <property name="text">
          <string>Do not use this proxy for:</string>
         </property>
         <property name="buddy">
          <cstring>exceptionsItemViewWidget</cstring>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="exceptionsHorizontalLayout">
         <item>
          <widget class="Otter::ItemViewWidget" name="exceptionsItemViewWidget">
           <property name="headerHidden">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="exceptionsVerticalLayout" stretch="0,0,0,1">
           <item>
            <widget class="QPushButton" name="addExceptionButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editExceptionButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeExceptionButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="exceptionsSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="exceptionsExampleLabel">
         <property name="text">
          <string>Example: domain.com, localhost, 127.0.0.1, 192.168.1.0/24</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::FilePathWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/FilePathWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::LineEditWidget</class>
   <extends>QLineEdit</extends>
   <header>src/ui/LineEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Otter::ProxyPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::ProxyPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
